src/pages/profile-page/profile-page.css

```diff
@@ -51,6 +51,12 @@
     display: flex;
     flex-direction: column;
     cursor: pointer;
+    background: var(--background-grey);
+    padding: 1rem;
+    border-radius: 999px;
```

Comment:
any specific reason to use px here? if not this can be changed to rem for CSS consistency

Reply:
Done